### DISCUSSION TITLE
Setting multiple theme options for a site

### DISCUSSION THREAD

**Discussion by Brian-Pob** [tailwindlabs/tailwindcss#10777] [Help]
Hello. I'm new to Tailwind. I read through the docs and noticed that Tailwind makes it pretty easy to set a dark mode style on a site. 

Is it possible to create and set multiple theme modes? For example, I'll have default, dark, red, blue, green, etc.

Ideally, I would set the styles similar to dark mode so I could just do `bg-white-400 dark:bg-gray-400 red:bg-red400 ...`

Thanks.

**Comment by wongjn** [comment]
It is possible to create and set multiple theme modes, [use a Tailwind plugin to add your own variants](https://tailwindcss.com/docs/plugins#adding-variants).

**Comment by andrepichardo** [comment]
Yes! It is possible, check this article on [How to Add Tailwind CSS Theme Variations](https://morioh.com/p/b3221f3f3d05), by just installing a simple plugin and following some steps.

Hope it helps!
